modules/ROOT/pages/fd-deploy-app-from-rm.adoc

```diff
@@ -1,15 +1,18 @@
 = Deploy a Mule Application from Runtime Manager
 
-It might be the case that you are not allowed to deploy an application directly from Flow Designer. Your organization might require someone with the proper authority to deploy applications from Runtime Manager. You can share with that person a copy of your project as a `.jar` file to deploy your application with.
+It might be the case that you are not allowed to deploy an application directly from Flow Designer.
+Your organization might require someone with the proper authority to deploy applications from Runtime Manager.
+You can share with that person a copy of your project as a `.jar` file to deploy your application with.
 
 == Procedure
 
 . In Flow Designer, follow these steps:
 .. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Package*.
+.. Export your project by clicking the dots next to the Test button and select *Export*.
+.. Select *Export project sources* to export just your project source files, or
```

Comment:
@mcherem why in this section of the documentation would we even talk about the **Export project sources** option, especially when we don't provide additional guidance to the user on when to use which option. i suggest we kill this line.

Reply:
or if this is the only place where we talk about the Export feature, then we need to explain in greater detail why someone would want to select this option. I would like to ask that you add
- **what it is for** (ie. when a user wants to export the project for the purposes of Importing it to Studio to take advantage of some of the advanced features available there, or if they want to collaborate with another someone else working on a Flow Designer project, and want to make it available for importing it there) 
- **what is the output** (i.e. what we exclude/include in this option vs the other option)

Reply:
Rewritten to reflect comments.